### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by clasqm**
Minor language edit

### CODE REVIEW

**Comment by deavmi** [comment]
Is there a reason for ignoring all files that end in `.html`?

**Review by deavmi — CHANGES_REQUESTED** [review]
* Undo the changes of the .gitignore unless there is a good reason for this.
* In the README add a `a` after the "Why not just write [here] "
* We do not need 2 extra lines after the end of the `README.md` in the root of the repo.
* Remove the License info from the document.
* Or the markdown editor you used.

Make these changes, I will do a final check and then everything should be fine for a merge.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,15 @@ Below is a list of documents related to all the software that comes pre-installe
...
+
+#### Utilities
+1. [**TinyTim**](docs/3rdPartySoftware/Utilities/TinyTim)
+	* Graphical front-end for tinyurl.com.
+
+
 ## License
 
 Creative Commons Attribution-ShareAlike 4.0 International
+
```
Comment (deavmi) [inline README.md]: Remove this.

**Comment by deavmi** [comment]
See if you can style the page like this too: https://github.com/deavmi/HaikuDocs/blob/en/TEMPLATE.md

**Comment by clasqm** [comment]
OK, I see where you are going with this.

_in the README add a a after the "Why not just write [here] "_

If that's the way you want it, but it is grammatically incorrect. "Documentation" is not a countable noun, you don't say "one documentation, two documentations".

I've deleted the gitignore. That was just to stop committing the htmls I generate to see what the final product looks like, in case I forgot to delete them on my offline branch. But I see now that that would then end up in the whole project. Sorry, I'll just have to be careful.

_We do not need 2 extra lines after the end of the README.md in the root of the repo._
done
_Remove the License info from the document._
done
_Or the markdown editor you used._
done

OK, that should help me in future contributions, now that I know what you would like to see here. The two of us may just be the entire South African Haiku Users Group, so we must stick together! ;-)

**Comment by deavmi** [comment]
Gonna merge soon tonight. Just wrote NBTs early this morning so I need some sleep. Thanks for the great work!

**Comment by deavmi** [comment]
Oh you can remove the `a` then again. Sorry about that grammar mistake.

**Comment by deavmi** [comment]
Try style it according yo CONTRIBUTING.md

**Review by deavmi — APPROVED** [review]

**Comment by deavmi** [comment]
Actually all is fine.

**Comment by deavmi** [comment]
We must definately stick toghether. The Haiku User Group in ZA is small. I really think this project can be successful as it is on GitHub and it is a platform that makes contributing really easy. Also Markdown is a great markup language to be doing this type of project in.

This is a side project of mine but when I am not busy with other work I will definately be working on this.

**Comment by clasqm** [comment]
I'm finding that I spend half an hour writing documentation, and a whole day fighting with GitHub. So my time will be better spent if I write up everything I want to add and then give it another shot. See you in a week or two.

**Comment by deavmi** [comment]
Thanks for the hard work and contiued hard work sir. I appreciate it. I will be busy with exams in those weeks but that's fine. I can then review the changes/additions in my spare time (which I know I will have - it's matric (not hell)).

**Comment by deavmi** [comment]
Also, sounds like a good plan to just write a whole chunk of documents and then commit and push them so that we basically have a day to review the changes and make tweaks if need be so.

Your first contribution worked out just like that; it was __stellar__ .

**Comment by deavmi** [comment]
Also, if you could just do this file move please, https://github.com/deavmi/HaikuDocs/issues/4, that would be great ;).

**Comment by deavmi** [comment]
One more thing. Don't add images to your next commit (do take screenshots) and add the image markdown code for where the image is to be but don't upload them. I want to investigate image hosting so that this repo doesn't become too large before we continue with screenshots.

**Comment by clasqm** [comment]
I deleted that branch after you merged. Now I need to resync my main branch
to yours before I can do anything else but WTH is the button for that? I am
starting to hate Github.

**Comment by deavmi** [comment]
Have you cloned the repo using `git`?

Make a backup first just incase.

If so then surely you can `git pull`?

**Comment by deavmi** [comment]
Tell me how that goes then we can see what to do next.

**Comment by deavmi** [comment]
Seeing that you were working in a seperate branch then updating the main branch should be as easy as `git pull` then you can make a new branch off of that? And put your changes into there and commit and push that to your repo and make a PR to merge from that branch to the `en` branch in my repo.

**Comment by deavmi** [comment]
GitHub is alright but it is only a frontend (only good feature - not that it is mandatory - is PRs).

You using `git` on the command-line?

**Pull request merged**